packit/config/aliases.py

```diff
@@ -46,6 +46,13 @@
     "s390x",
     "x86_64",
 ]
+
+DEPRECATED_TARGET_MAP = {"centos-stream": "centos-stream-8"}
+for k, v in list(DEPRECATED_TARGET_MAP.items()):
+    DEPRECATED_TARGET_MAP.update(
+        map(lambda x: ("-".join([k, x]), "-".join([v, x])), ARCHITECTURE_LIST)
```

Comment:
I usually find list/dict comprehension easier to read than `map(lambda(...))
```suggestion
        [(f"{k}-{x}", f"{v}-{x}") for x in ARCHITECTURE_LIST]
```

or even the whole `for` loop can be replaced with one comprehension:
```python
DEPRECATED_TARGET_MAP = {
    f"{k}-{x}": f"{v}-{x}"
    for k, v in DEPRECATED_TARGET_MAP.items()
    for x in ARCHITECTURE_LIST
}

```